### PR TITLE
Add Entrypoints

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

**Comment by martindurant** [comment]
Ref #394

File: intake/cli/client/subcommands/drivers.py [hunk 1]
```diff
@@ -76,20 +81,34 @@ def _list(self, args):
                   if k not in all_drivers and k not in drivers_by_name}
 
         print("Direct:", file=sys.stderr)
-        for name, cls in direct.items():
+        none = True
```
Comment (danielballan) [inline intake/cli/client/subcommands/drivers.py]: Have you considered the pattern:

```python
for ...:
    ...
else:
    print("<none>")
```

The `for...else` usage is slightly obscure, which is a downside of this approach, but it is more succinct and saves you the `none` variable.

Comment (martindurant) [reply intake/cli/client/subcommands/drivers.py]: I did exactly that, but didn't get the right behaviour, especially below, where not every element of `for` counts.

Comment (danielballan) [reply intake/cli/client/subcommands/drivers.py]: Ohhh. Got it. Sorry for the noise, carry on! :-)

**Pull request merged**